### ISSUE TITLE
Student Issues Cloning Lesson | C# Web Basics

### ISSUE THREAD

**Issue by vstankatz** [epicodus-lessons/section-2-to-do-list-csharp-net6#1]
I had a couple students inform me this morning that one of the GitHub example repositories listed in the LHTP lessons for the week no longer allows students to clone the project at specific branches. [This is the GitHub repository](https://github.com/epicodus-lessons/section-2-to-do-list-csharp-net6/tree/8_saving_OWO_and_interface_update) linked to, in this example the students are linked to branch `8_saving_OWO_and_interface_update` but when cloning the project they only have access to `1_mvc_setup`. After meeting with a student I tried to have them follow the same steps I use to view different branches:

- Run `git remote -v` to find the name of the project, in this case `origin` when cloned.
- Run `git branch` to view the current available branches, in this case only `1_mvc_setup`.
- Run `git fetch origin 8_saving_OWO_and_interface_update`, to get the desired branch.
- Run `git pull origin 8_saving_OWO_and_interface_update`, this should have pulled down the new branch into its own separate branch but instead through rebase and ff errors. 

Students were able to get the desired branch if they used the `Download Zip` option for the project on GitHub but no other branches were able to be cloned.

tl;dr:
Expected behavior from students: Ability to clone down program at various branch instances like with all other past example repos.
Behavior students are getting: Only the default branch is able to be cloned, unable to pull others.
